desktop/ui/src/main/java/org/datacleaner/widgets/tree/SchemaTree.java

```diff
@@ -108,13 +108,14 @@
     private final WindowContext _windowContext;
     private final AnalysisJobBuilder _analysisJobBuilder;
     private final InjectorBuilder _injectorBuilder;
+    private boolean _includeLibraryNode = true;
 
     private DatastoreConnection _datastoreConnection;
     private String _searchTerm = "";
 
     @Inject
     protected SchemaTree(final Datastore datastore, @Nullable AnalysisJobBuilder analysisJobBuilder,
-            WindowContext windowContext, InjectorBuilder injectorBuilder) {
+            WindowContext windowContext, InjectorBuilder injectorBuilder) throws Exception {
```

Comment:
I cannot see why the constructor should suddenly start throwing Exception? Nothing seems to have changed.